operations/research-and-development/product/pm-onboarding.md

```diff
@@ -0,0 +1,240 @@
+
+### Training focus:
+
+* Introduction to our Customer Obsession Principle
+* Introductions to team members through 1-1s
+  * Use introductions as learning opportunities to understand different parts of the company \(ask questions!\)
+* Introduction to product
+* Introduction to our “Fast, Obvious, Forgiving” design principles
+* Introduction to key resources
+* Make your first contribution on GitHub
+
+### Activities:
+
+Customer Obsession:
+* Join and introduce yourself in these channels, ask to join any and all customer calls - see if you can get a couple scheduled this week:
+  * [Product Management](https://community.mattermost.com/private-core/channels/product-management)
+  * [GTM:PM Engagement](https://community.mattermost.com/private-core/channels/csmpm-engagement) \(Stu, Jenn\)
+  * [Sales Discussion](https://community.mattermost.com/private-core/channels/sales-discussion) \(Lance, Steve, Richard Pidgeon)
+* Join and review the topics and posts in these channels to get a better understanding of our customers
+  * [Customer Feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
+  * [Customer Support](https://community.mattermost.com/private-core/channels/community)
+* Begin watching training videos on [academy.mattermost.com ](https://academy.mattermost.com/)
+* Begin reading [docs.mattermost.com](https://docs.mattermost.com/)
+
+Team:
+* Meet the rest of team leads \(Developers, Designer, QA\) and join team meetings
+  * Ask questions to understand development, design, QA processes
+* Meet with Amy \(Release Manager\) and read the release process doc
+* Meet with Ian (CEO) \[[https://github.com/it33/readme/blob/master/README.md](https://github.com/it33/readme/blob/master/README.md/)\). This will be scheduled for you
+* Meet other Product Managers \(Jason Blais, Eric Sethna, Katie Wiersgalla, Aaron Rothschild, Ian Tao, Chen-I Lim\)
+  * Learn about their areas of ownership
+* Participate in PM, R&D, and Customer Obsession Meetings
+* Jira
+  * Sign up to Jira and post in the “Jira configuration” channel to be added to internal team
+  * Get Jira training from PM buddy
+    * [Mattermost Platform](https://mattermost.atlassian.net/projects/MM/issues/MM-10977?filter=allopenissues)
+* Make your first edit and pull request on GitHub
+  * Update documentation or product help text
+* Small contributions
+  * File a bug in Jira
+  * Test a new feature on a test server and suggest an improvement
+  * Leave feedback on a design proposal
+
+Readings:
+* The Advantage:
+  * [https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view](https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view)
+* The Five Dysfunctions of a Team: A Leadership Fable
+  * [https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing](https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing)
+
+## First 30 days
```

Comment:
This is a lot. I'd like to see three outcomes for a PM 
1. Learn more about our users and customers, either via direct conversations with them and our team (which you have), or via quantitative data (e.g. via telemetry)
2. Ship their first feature. This can be a small enhancement, but would let them get exposed to the feature set, release process, and more.
3. Take ownership of KPIs and roadmap by end of day 30 -- depending on seniority and scope, this could be a subset of the roadmap they'll eventually own, e.g. roadmap for compliance, or for Desktop App. 

Let me know what you think?

Reply:
I personally think shipping first feature and taking ownership of KPIs and roadmap is too aggressive.  In my mind, they should be immersing themselves into learning and getting the flow and cadence of what we are doing and why.  I'd be hesitant to push them to get full ownership until they have the first 30 days under their belt.  

I do have full ownership in days 30-60, which takes them into building a feature from the start.